package.json

```diff
@@ -2,11 +2,15 @@
   "name": "room.inlive.app",
   "version": "0.1.0",
   "private": true,
+  "type": "commonjs",
```

Comment:
We don't need to specify the type actually. By default, when type is not set, it is treated as commonjs. See https://nodejs.org/api/packages.html#packagejson-and-file-extensions